### PR TITLE
2024.03-LTS updates

### PR DESCRIPTION
**Pull request by MalarsriA**

### CODE REVIEW

File: development/docker-compose/.env [hunk 1]
```diff
@@ -1,2 +1,2 @@
-BB_VERSION=2023.10.5
-STREAM_VERSION=3.63.0
+BB_VERSION=2024.03-LTS
+STREAM_VERSION=4.0.0
```
Comment (miladbb) [inline development/docker-compose/.env]: Does stream version 4.0.0 work with the BBBOM version?

Comment (MalarsriA) [reply development/docker-compose/.env]: Yes, i saw it is used in other libraries, https://github.com/Backbase/mastercard-openbanking/blob/main/backbase-environment/.env

Comment (miladbb) [reply development/docker-compose/.env]: When I run it with version 4.0.0, it does not create the products as expected

Comment (MalarsriA) [reply development/docker-compose/.env]: ok i am reverting back it with 3.63.0. will update with 2024.04

**Review by miladbb — APPROVED** [review]

**Pull request merged**